README.md

```diff
@@ -33,7 +33,7 @@ Which need the `@sdc/shared` package to have been built.
 To start the server run
 
 ```bash
-yarn server start
+NODE_OPTIONS="--max-old-space-size=80000" AWS_PROFILE=membership PORT=8082 yarn server start
```

Comment:
change here as well?
https://github.com/guardian/support-dotcom-components/blob/main/packages/server/src/server.ts#L482

Reply:
ok done - I was just worried that is the one used in PROD and I would not be able to deploy!

Reply:
I think I need to change here too if I do that? https://github.com/guardian/support-dotcom-components/blob/4519b5c6a42b81a242d3283aedd38e5afe24f8c6/cdk/lib/dotcom-components.ts#L78

Reply:
ah good point! I think you can ignore my above comment and just use 3030 in PROD!